bigtable/admin.go

```diff
@@ -298,6 +352,15 @@ func (ac *AdminClient) CreateTableFromConf(ctx context.Context, conf *TableConf)
 		tbl.ChangeStreamConfig = &btapb.ChangeStreamConfig{}
 		tbl.ChangeStreamConfig.RetentionPeriod = durationpb.New(conf.ChangeStreamRetention.(time.Duration))
 	}
+
+	var err error
+	if conf.AutomatedBackupConfig != nil {
+		tbl.AutomatedBackupConfig, err = toAutomatedBackupConfigProto(conf.AutomatedBackupConfig)
+		if err != nil {
+			return err
+		}
+	}
```

Comment:
I think this would be nicer:

```go
if conf.AutomatedBackupConfig != nil {
  if proto, err := toAutomatedBackupConfigProto(conf.AutomatedBackupConfig); err != nil {
    return err
  } else {
    tbl.AutomatedBackupConfig = proto
  }
}
```

Reply:
Thanks, updated